README.md

```diff
@@ -47,7 +47,24 @@ zigup stores each compiler in a global "install directory" in a versioned subdir
 
 zigup makes the zig program available by creating an entry in a directory that occurs in the `PATH` environment variable.  On posix systems this entry is a symlink to one of the `zig` executables in the install directory.  On windows this is an executable that forwards invocations to one of the `zig` executables in the install directory.
 
-Both the "install directory" and "path link" are configurable through command-line options `--install-dir` and `--path-link` respectively.  On posix systems the default "install directory" follows the [XDG basedir spec](https://specifications.freedesktop.org/basedir-spec/latest/#variables), ie. `$XDG_DATA_HOME/zigup` or `$HOME/.local/share/zigup` if `XDG_DATA_HOME` environment variable is empty or undefined.
+Both the "install directory" and "path link" are configurable through command-line options `--install-dir` and `--path-link` respectively, as well as in your configuration file.
+
+# Configuration
+
+Configuration happens from two places: command-line options and the configuration file. For each configuration parameter, command-line options always override the configuration file. If neither are set a default value is used.
+
+The configuration file is in ZON (Zig Object Notation) format. The default path for this configuration file is `~/.config/zigup.zon` for posix, and `TODO` for Windows. This can be overrided using the `--config` command-line option.
+
+Any fields not set in your configuration file will use default values. For example, the following configuration file only sets the `install_dir` parameter, and leaves the rest to their default values:
```

Comment:
I don't think an example is needed to demonstrate how "default values" work.  But...having an example configuration file is good.

Reply:
Can remove this, especially since the `show-config` command can function as an example configuration.